### DISCUSSION TITLE
Add view background settings: transparency, size, aligment, tile

### DISCUSSION THREAD

**Discussion by marcinbauer-ohf** [home-assistant/frontend#22931] [Dashboard] [answered]
This design addresses a proposal to add dashboard view background settings. There is already a contribution made by @karwosts, but we want to expand that PR [https://github.com/home-assistant/frontend/pull/22643](url) with additional options.

> [!NOTE]
> Detailed notes and previews can be found in the Figma file.

## Design & Notes
https://www.figma.com/design/3yQP7fM0n5ryMcx8d9TTLq/HA-15?node-id=2-110&node-type=section&t=EUe1HaGVLR3iNLu3-11

_- Home Assistant (Nabu Casa) Product Team_

**Comment by silamon** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/23133
